src/test/java/com/sigma/cinematicketpro/mapper/impl/MovieMapperImplTest.java

```diff
@@ -0,0 +1,51 @@
+package com.sigma.cinematicketpro.mapper.impl;
+
+import com.sigma.cinematicketpro.TestUtils;
+import com.sigma.cinematicketpro.dto.MovieDto;
+import com.sigma.cinematicketpro.entity.Movie;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.modelmapper.ModelMapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class MovieMapperImplTest {
+    @Mock
+    private ModelMapper modelMapper;
+    @InjectMocks
+    private MovieMapperImpl sut;
+
+    @Test
+    void shouldCallModelMapperMap_WhenToDto() {
+        Movie movie = TestUtils.getValidMovies().get(0);
+        MovieDto expectedMovieDto = TestUtils.getMovieDto(movie);
+
+        when(modelMapper.map(movie, MovieDto.class)).thenReturn(expectedMovieDto);
+
+        var actualMovie = sut.toDto(movie);
+
+        assertThat(expectedMovieDto).isEqualTo(actualMovie);
+
+        verify(modelMapper, times(1)).map(movie, MovieDto.class);
+    }
+    @Test
+    void shouldCallModelMapperMap_WhenToEntity() {
+        Movie expectedMovie = TestUtils.getValidMovies().get(0);
```

Comment:
looks like you need getValidMovie helper method in TestUtils

Reply:
added getValidMovie method in TestUtils